packages/actor-http-fetch/test/ActorHttpFetch-test.ts

```diff
@@ -338,5 +338,26 @@ describe('ActorHttpFetch', () => {
       expect(fetch).not.toHaveBeenCalled();
       expect(customFetch).toHaveBeenCalled();
     });
+
+    it('should work with a timeout', async() => {
+      await expect(actor.run({
+        input: <Request> { url: 'https://www.google.com/' },
+        context: new ActionContext({ [KeysHttp.httpTimeout.name]: 100 }),
+      })).resolves.toMatchObject({ status: 200 });
+    });
+
+    it('should abort properly with a timeout', async() => {
```

Comment:
This test is a bit fragile and very slow because it relies on timeout behavior.
I am not sure how to write it differently without a lot of dependency injection.

Reply:
We can make it faster using Jest's timer mocks: https://jestjs.io/docs/timer-mocks